rotkehlchen/tests/unit/test_aave.py

```diff
@@ -97,7 +98,7 @@ def test_aave_v1_events_stats(
     aave = Aave(
         ethereum_inquirer=ethereum_inquirer,
         database=database,
-        premium=Premium(rotki_premium_credentials),
+        premium=Premium(rotki_premium_credentials, username),
```

Comment:
in all these places. There is many files. Please make sure they are all checked.